docs/source/en/quicktour.mdx

```diff
@@ -502,11 +502,11 @@ All models are a standard [`tf.keras.Model`](https://www.tensorflow.org/api_docs
 
    ```py
    >>> def tokenize_dataset(dataset):
-   ...     return tokenizer(dataset["text"])
```

Comment:
Running `make style` adds two lines of empty space after the `tokenize_dataset` function, which caused it to render on the docs like this:

<img width="776" alt="Screen Shot 2022-09-07 at 11 09 22 AM" src="https://user-images.githubusercontent.com/59462357/188949803-59853086-4d31-473e-bd7a-c138da9b8e94.png">

If I remove these empty lines, then the `check_code_quality` test fails. Any idea how to workaround this?

Reply:
Could you try add one empty line, and see if `make style` still change it to 2 empty lines?

Reply:
Otherwise, probably just separate the function definition and their usage into 2 difference blocks of doc examples.